docs/source-2.0/guides/smithy-cli/cli_installation.rst

```diff
@@ -0,0 +1,423 @@
+
+    .. tab:: Manual (x64)
+
+        First, retrieve the latest smithy installation from the `Smithy GitHub releases`_.
+
+        .. code-block:: powershell
+            :caption: powershell
+            :substitutions:
+
+            PS > (mkdir smithy-install\smithy -Force) -and
+                (wget |release-uri|/|windows-tar| -outfile smithy-install\|windows-tar|)
+
+        Now, untar the installation in the newly created directory and run the installer(``install.bat``).
+
+        .. important:: The following command requires running powershell with administrator access in order to
+            install the CLI into the default location (``<Letter>:\Program Files\Smithy``). Alternatively, you
+            may specify your own path, which can mitigate permissions issues when not running as administrator.
+
+        .. code-block:: powershell
+            :caption: powershell
+            :substitutions:
+
+            PS > (tar -xf smithy-install\|windows-tar| -C smithy-install\smithy) -and
+                (smithy-install\smithy\install)
+
+        Follow the installer prompts accordingly to complete the installation.
+
+        After the install completes, you can now run ``smithy --help`` to verify your installation.
+        You may also delete all the files downloaded for the installation (``smithy-install\``).
+
+
+.. _cli_verification:
+
+Verification (Optional)
+-----------------------
+
+The Smithy CLI distributables are PGP-signed and hashed before they are released.
+You should verify the signatures and hashes of the archives to ensure the validity and integrity
+of what you are downloading. You can follow the steps below to do so for your given platform.
+
+Before proceeding, please make sure you have both the GnuPG utility(``gpg``) and a sha256-checksum
+utility (``sha256sum``) installed on your system. We'll be using these to perform the verification.
+
+.. note:: This is only applicable to manual installations - Homebrew verifies checksums before install.
+
+.. tab:: MacOS
+
+    Download the detached signature file(``.asc``) and the sha256-checksum file(``.sha256``) corresponding to
+    the tarball (which you already downloaded) from the `Smithy GitHub releases`_.
+
+    .. tab:: Manual (x86)
+
+        .. code-block:: sh
+            :caption: /bin/sh
+            :substitutions:
+
+            $ curl -L '|release-uri|/|mac-tar|.{asc,sha256}' -o 'smithy-install/|mac-tar|.#1'
+
+        Verify the checksum of the tarball using the `sha256sum` utility.
+
+        .. code-block:: sh
+            :caption: /bin/sh
+            :substitutions:
+
+            $ (cd smithy-install && sha256sum -c |mac-tar|.sha256)
+
+        Now, retrieve our `public PGP key`_ from the repository, and import it into your key-chain.
+
+        .. code-block:: sh
+            :caption: /bin/sh
+            :substitutions:
+
+            $ curl -L |smithy-pub-key-uri| -o smithy-install/smithy.asc && gpg --import smithy-install/smithy.asc
```

Comment:
This one `https://raw.githubusercontent.com/awslabs/smithy/main/smithy.asc` currently returns ~404~ (just noticed that is added in this PR), but having said that, we rotate the key every 3 months, not sure what to do about that if for whatever reason they get a copy that was signed with a key that has been already rotated. Also now we need to make sure that as part of the key rotation process we update it here as well.

Reply:
Hmmm... good point. Maybe we shouldn't put it in the repo then, and instead include it in the release assets themselves.